pkg/mover/chart.go

```diff
@@ -236,17 +285,116 @@ func namespacedPath(fullpath, chartName string) string {
 }
 
 /*
-Move executes the Chart relocation which includes
+  Move performs the relocation.
+
+A regular move executes the Chart relocation which includes
 
 1 - Push all the images to their new locations
 
 2 - Rewrite the Helm Chart and its subcharts
 
 3 - Repackage the Helm chart as toChartFilename
+
+A save to an offline tarball bundle will:
+
+1 - Drop all images to disk, with the original chart (unpacked) and hints file
+
+2 - Package all in a single compressed tarball
 */
 func (cm *ChartMover) Move() error {
+	if cm.targetOfflineTar != "" {
+		return cm.saveOfflineBundle()
+	}
+	return cm.moveChart()
+}
+
+func (cm *ChartMover) saveOfflineBundle() error {
 	log := cm.logger
 
+	tarPath, err := os.MkdirTemp("", "offline-tarball-*")
```

Comment:
maybe add relok8s to the prefix?

Reply:
I'd rename tarPath to bundleWorkDir since in there you will not have the tarball but a working directory that then it will be compressed and stored as a tarball somewhere else (targetOfflineTarPath)

Reply:
Agreed, renamed.